src/ContainerLogs.jsx

```diff
@@ -149,7 +139,18 @@ class ContainerLogs extends React.Component {
             }
             // First 8 bytes encode information about stream and frame
             // See 'Stream format' on https://docs.docker.com/engine/api/v1.40/#operation/ContainerAttach
-            this.view.writeln(data.slice(8));
+            while (data.byteLength >= 8) {
+                // split into frames (size is the second 32-bit word)
+                const size = data[7] + data[6] * 0x100 + data[5] * 0x10000 + data[4] * 0x1000000;
+                const frame = data.slice(8, 8 + size);
+                // old podman versions just have CR endings, append NL then
+                if (frame[size - 1] === 13)
+                    this.view.writeln(frame);
```

Comment:
This added line is not executed by any test.

Reply:
Right, this only happens on ubuntu-22.04.